client/state/sites/plans/test/selectors.js

```diff
@@ -226,10 +227,112 @@ describe( 'selectors', () => {
 					}
 				}
 			};
-			const discountPrice = getPlanDiscountPrice( state, 77203074, 'silver', true );
+			const discountPrice = getPlanDiscountedRawPrice( state, 77203074, 'silver', true );
 			expect( discountPrice ).to.equal( null );
 		} );
 	} );
+
+	describe( '#getPlanRawDiscount()', () => {
+		it( 'should return a raw discount', () => {
+			const plans = {
+				data: [ {
+					currentPlan: false,
+					productSlug: 'gold',
+					rawPrice: 299,
+					rawDiscount: 0
+				}, {
+					currentPlan: false,
+					productSlug: 'silver',
+					rawPrice: 199,
+					rawDiscount: 0
+				}, {
+					currentPlan: true,
+					productSlug: 'bronze',
+					rawPrice: 99,
+					rawDiscount: 100
+				} ]
+			};
+
+			const state = {
+				sites: {
+					plans: {
+						77203074: plans
+					}
+				}
+			};
+
+			const planRawDiscount = getPlanRawDiscount( state, 77203074, 'bronze' );
+
+			expect( planRawDiscount ).to.equal( 100 );
+		} );
+
+		it( 'should return a monthly raw discount', () => {
+			const plans = {
+				data: [ {
+					currentPlan: false,
+					productSlug: 'gold',
+					rawPrice: 299,
+					rawDiscount: 0
+				}, {
+					currentPlan: false,
+					productSlug: 'silver',
+					rawPrice: 199,
+					rawDiscount: 0
+				}, {
+					currentPlan: true,
+					productSlug: 'bronze',
+					rawPrice: 99,
+					rawDiscount: 100
+				} ]
+			};
+
+			const state = {
+				sites: {
+					plans: {
+						77203074: plans
+					}
+				}
+			};
+
+			const planRawDiscount = getPlanRawDiscount( state, 77203074, 'bronze', true );
```

Comment:
I'm not too familiar with this part of Calypso, but it took some looking to figure out what the `true` here was doing. I know this is pre-existing code, but could we use an options parameter here for clarity? eg:

```js
const discountPrice = getPlanDiscountedRawPrice( state, 77203074, 'bronze', { isMonthly: true } );
```